src/builders/libraryBuilder.ts

```diff
@@ -75,7 +75,37 @@ export class LibraryBuilder {
     return resources;
   }
 
-  buildLibrary(): string {
+  createLibraryResource(): R4.ILibrary {
+    const { name, version } = this.ig;
+    logger.info('generating FHIR Library');
+    return {
+      resourceType: 'Library',
+      id: name,
+      name,
+      version,
+      type: {
+        coding: [
+          {
+            system: 'http://terminology.hl7.org/CodeSystem/library-type',
+            code: 'logic-library'
+          }
+        ]
+      },
+      status: R4.LibraryStatusKind._draft,
+      content: [
+        {
+          contentType: 'text/cql',
+          url: `${name}.cql`
```

Comment:
We may need to think about how/if we provide this information as a url.  From the spec: 

" Relative URLs are interpreted relative to the service url, like a resource reference, rather than relative to the resource itself."  

 Just supplying the name of the file will probably be considered a relative reference which would then need to be resolvable to the base url from which the Questionnaire was retrieved.   We may want to consider base64 encoding the files and adding as data attribute on the content objects.

Reply:
Ok, I switched to using `data` now. Had to use a non-null assertion for the library content to push the ELM onto it later hope that's fine haha